frontend/src/components/Overlay/OverlayHeader.tsx

```diff
@@ -0,0 +1,30 @@
+import OverlayCloseIcon from './OverlayCloseIcon';
+
+import './OverlayHeader.css';
+
+interface OverlayHeaderProps {
+	closeOverlay: () => void;
+	heading: string;
+	iconColor?: string;
+}
+
+function OverlayHeader({
+	closeOverlay,
+	heading,
+	iconColor,
+}: OverlayHeaderProps) {
+	const iconFill: string = iconColor ?? '#000';
```

Comment:
In fact, rather than declaring this as a separate const, better to use a default value for the prop itself:

```
function OverlayHeader({
  closeOverlay,
  heading,
  iconColor = '#000',
}: OverlayHeaderProps) {
```